yew/src/virtual_dom/vtag.rs

```diff
@@ -24,7 +24,7 @@ cfg_if! {
         use std::ops::Deref;
         use wasm_bindgen::JsCast;
         use web_sys::{
-            Element, HtmlInputElement as InputElement, HtmlTextAreaElement as TextAreaElement, Node,
+            Element, HtmlInputElement as InputElement, HtmlTextAreaElement as TextAreaElement, Node, HtmlButtonElement
```

Comment:
@jstarry accordint to travis-ci check: [unresolved import](https://travis-ci.com/github/yewstack/yew/jobs/323468442#L507) which is strange, because it should work

Reply:
Which import does it say is unresolved?

Reply:
I've created a pull request in your repository which I think should solve the problem.
To use a feature from `web-sys` you need to activate it (in the `Cargo.toml` configuration).